packages/smooth_app/lib/widgets/smooth_product_carousel.dart

```diff
@@ -269,7 +266,7 @@ class _SearchCardTagLine extends StatelessWidget {
                     style: TextStyle(
                       fontSize: 18.0,
                       height: 1.5,
-                      color: themeProvider.color,
+                      color: Theme.of(context).colorScheme.primary,
```

Comment:
Strange to see:
* a text written in `primary` color (should be `onPrimary`)
* a text written in a color without a background color

Reply:
It was `themeProvider.color` before which was also the primary color, so I just left it this way